package.json

```diff
@@ -7,13 +7,20 @@
     ]
   },
   "devDependencies": {
-    "@redwoodjs/core": "^0.2.4"
+    "@redwoodjs/core": "^0.31.2"
   },
   "eslintConfig": {
-    "extends": "@redwoodjs/eslint-config"
+    "extends": "@redwoodjs/eslint-config",
+    "root": true
   },
   "engines": {
-    "node": ">=12",
+    "node": ">=14",
```

Comment:
Updated node version could be a problem for production?

Reply:
Should be fine!